From 2df5d760d88810da16a597c3f95ca8ef3abef2e7 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Mon, 1 Jul 2019 11:19:21 +0800
Subject: [PATCH] disable chronyd

There is no chronyd services provides, the systemd in Yocto
has its own ntp

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 pyanaconda/core/configuration/system.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyanaconda/core/configuration/system.py b/pyanaconda/core/configuration/system.py
index 9b7ef96..9dfbadd 100644
--- a/pyanaconda/core/configuration/system.py
+++ b/pyanaconda/core/configuration/system.py
@@ -99,7 +99,7 @@ class SystemSection(Section):
     @property
     def can_set_time_synchronization(self):
         """Can we run the NTP daemon?"""
-        return self._is_boot_iso or self._is_booted_os
+        return False
 
     @property
     def can_activate_keyboard(self):
-- 
2.7.4

